Program.cs

```diff
@@ -3,17 +3,230 @@
+            Write (letter + sSpaces);
+         }
+         ResetColor ();
+         WriteLine ();
+      }
+
+      /// <summary>Prints the message in corresponding color</summary>
+      /// <param name="color">Color of the message</param>
+      /// <param name="message">The message to be printed</param>
+      static void PrintMessage (EColor color, string message) {
+         SetCursorPosition (sKeyBoardColumn, sRow * 10);
+         ChangeConsoleColor (color);
+         Write (color == Yellow ? $"{message,33}" : $"{message,35}");
+         ResetColor ();
+      }
+
+      /// <summary>Changes the ForegroundColor as required</summary>
+      /// <param name="color">Color to which the ForegroundColor must be changed</param>
+      static void ChangeConsoleColor (EColor color) {
+         ForegroundColor = color switch {
+            Green => ConsoleColor.Green,
+            DarkGray => ConsoleColor.DarkGray,
+            Yellow => ConsoleColor.Yellow,
+            White => ConsoleColor.White,
+            Blue => ConsoleColor.Blue,
+            _ => ConsoleColor.Black,
+         };
+      }
+
+      /// <summary>This method reads the given file and stores the lines in the corresponding list</summary>
+      /// <param name="file">File name</param>
+      static void ReadFile (string file) {
+         var stream = Assembly.GetExecutingAssembly ().GetManifestResourceStream ($"Training.data.{file}.txt");
+         using var reader = new StreamReader (stream);
+         while (!reader.EndOfStream)
+            (file == "puzzle" ? sWords : sValidWords).Add (reader.ReadLine ());
+      }
+      #endregion
+
+      #region Enum --------------------------------------------------
+      public enum EColor { DarkGray, Green, Yellow, White, Black, Blue }
+      #endregion
+
+      #region Members -----------------------------------------------
+      static char[] mLetters = new char[5]; // Entry characters
+      static int mKeyColumn, mKeyRow, mLetterCount = 0; // Position at which a key is pressed.
+      static string mWordleWord, mUserWord; // Random word to be picked by computer, user typed word respectively.
+      static Dictionary<char, EColor> mKeyboard = new (); // Keyboard letters and their corresponding colors
+      #endregion
+
+      #region Private -----------------------------------------------
+      static char sDot = '\u00b7', sCircle = '\u25cc';
+      static string sSpaces = "    "; // 4 spaces
+      static List<string> sWords = new (); // Wordle words
+      static List<string> sValidWords = new (); // Valid English Words
+      static int sRow = 3, sColumn = WindowWidth / 2 - 10;  // For displaying the initial position of the game.
+      static int sKeyBoardRow = 23, sKeyBoardColumn = WindowWidth / 2 - 20; // Co-ordinates of the initial position of the keyboard.
+      static int sRowGap = 3, sColumnMaxLimit = WindowWidth / 2 + 10; // Lines between rows, max column limit of game area respectively.

```

Comment:
Computing WindowWidth/2 multiple times. Instead store in a variable and resuse it.